### DISCUSSION TITLE
how to upgrade from 1.6 to 1.8

### DISCUSSION THREAD

**Discussion by wguanicedew** [indigo-iam/iam#741] [Q&A]
We have an Indigo IAM service. Which is 1.6.0? how to upgrade it to 1.8? With directly upgrade to 1.8, it seems the database version is different and it will fail. Thanks.

**Comment by rmiccoli** [comment]
Hi, you have to upgrade first to IAM v1.7.2 as described in the release notes here https://github.com/indigo-iam/iam/releases/tag/v1.8.0 (_Upgrading_ section).

**Comment by wguanicedew** [comment]
The upgrade from 1.6 -> 1.7.2 -> 1.8 works. One thing to add, from 1.7.2 to 1.8, flyway requires new version of mariadb. The default mariadb on centos 7 is 5.5, which flyway doesn't support. After upgrading the db, all things are ok (We are doing the way to migrating IAM from Centos 7 to Alma 9. Currently we are just testing the migration to make sure it can be upgraded). Thanks.
